docs/source/conf.py

```diff
@@ -40,7 +39,7 @@
 
 # General information about the project.
 project = "aiidalab"
-copyright_first_year = "2020"
```

Comment:
I think this should remain `2020`. It is the "first year" not the current year.

Reply:
Ah, yes. I changed it back.